src/main/java/Tars.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+
+public class Tars {
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        // Opening speech.
+        System.out.println("    " + "------------------------------------------------------------");
+        System.out.println("    " + "Hello! I'm Tars.");
+        System.out.println("    " + "Ready to conquer the world? Or maybe just help with something smaller? What can I do for you?");
+        System.out.println("    " + "------------------------------------------------------------");
+
+        // Define an array to store tasks with a maximum capacity of 100 tasks, using the Task class to store data.
+        Task[] tasks = new Task[100];
+        int taskCount = 0;
+
+        // Waiting for user input.
+        Scanner scanner = new Scanner(System.in);
+        String input;
+
+        while (true)
+        {
+            // Read user input.
+            input = scanner.nextLine();
+
+            // Exit the program when the user inputs "bye".
+            if (input.equals("bye"))
+            {
+                System.out.println("    " + "------------------------------------------------------------");
+                System.out.println("    " + "Oh no, you're leaving already? Fine, be that way. But don't forget, I'll be right here waiting... forever!");
+                System.out.println("    " + "------------------------------------------------------------");
+                break;
```

Comment:
Could you maybe create 1 function for each of the commands (bye, mark, list...) and have those functions deal with the appropriate code? So as to not have an overly long main method.